.github/workflows/run_build_deploy.yaml

```
@@ -48,6 +48,56 @@ jobs:
       image-tag: latest
       env-name: dev
 
+  run-e2e-tests:
+    name: Run E2E Test Suite
+    needs: [ deploy-dev ]
```

Since this will make running E2E tests part of the "Build and Deploy" workflow, I think we should add `continue-on-error: true` to prevent it from failing the entire workflow when the tests fail. Especially since these tests tend to fail more than they succeed

```suggestion
    needs: [ deploy-dev ]
    continue-on-error: true
```